pymc/sampling/mcmc.py

```diff
@@ -144,6 +146,13 @@ def instantiate_steppers(
         if initial_point is None:
             initial_point = model.initial_point()
 
+        for rv in model.free_RVs:
```

Comment:
There's an interface for samplers to say they prefer a variable of a given type, it shouldn't happen here.